### DISCUSSION TITLE
How to disable the code being printed at the beginning?

### DISCUSSION THREAD

**Discussion by DevloperHS** [hemantapkh/PyBirthdayWish#15] [Q&A] [answered]
Hello, I followed your steps to run the program, but even without changing it is printing the codes rather than the art, How to fix it?

![Printing Error](https://user-images.githubusercontent.com/74095712/119135273-d39cf000-ba5b-11eb-8cf1-ea2dcdc35c38.PNG)

Please provide the fix

How to remove the code part?

**Comment by murshedjamilalif** [comment]
same problem bro......have you got any solution ???

**Comment by hemantapkh** [comment, marked as answer]
You can easily disable the code printing by changing `codePrint=False` in config.py file [here](https://github.com/hemantapkh/PyBirthdayWish/blob/1a57f4ae444e5390b4069e9d09645fd576e43892/config.py#L6).
@DevloperHS @murshedjamilalif

**Comment by murshedjamilalif** [comment]
tnx bro....it's working....

**Comment by murshedjamilalif** [comment]
after showing the texts ,,,,take your cursor on the last line and press
enter ,,,,the art will appear then
